hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -1789,12 +1789,10 @@ void compactIfNecessary(BaseHoodieWriteClient<?,I,?,O> writeClient, Option<Strin
       writeClient.compact(compactionInstantTime, true);
     } else if (metadataWriteConfig.isLogCompactionEnabled()) {
       // Schedule and execute log compaction with new instant time.
-      final String logCompactionInstantTime = metadataMetaClient.createNewInstantTime(false);
-      if (metadataMetaClient.getActiveTimeline().filterCompletedInstants().containsInstant(logCompactionInstantTime)) {
-        LOG.info("Log compaction with same {} time is already present in the timeline.", logCompactionInstantTime);
```

Comment:
is this case now handled within writeClient.schedule....  ? I peeked quickly.. writeClient delegates to the table obj

Reply:
Right now we are assuming the skew guarantees uniqueness across jvms, uniqueness within a jvm is guaranteed by `HoodieInstantTimeGenerator`. I'm not sure how this condition would return true currently. It mainly looks like it is copied from the compaction case where the the instant time can be manually generated to by just before the last inflight commit.